core/alsp_src/builtins/sio.pro

```diff
@@ -799,8 +808,36 @@ check_list_option(M,ML) :-
 	dmember(M,ML),
 	!.
 check_list_option(M,ML) :-
-	domain_error(stream_option,M,3).
+        domain_error(stream_option,M,3).
```

Comment:
pedantic, but maybe revert the whitespace change in `check_list_options` to get a cleaner diff.